qml/model/CallHistory.qml

```diff
@@ -139,38 +138,52 @@ Db8Model {
 
         if( endedVoiceCall.isIncoming ) {
             newCallItem.from = personDetails;
-            newCallItem.to.push ({ addr: "", service: "com.palm.telephony" });
+            newCallItem.to.push({ addr: "", service: "com.palm.telephony" });
             newCallItem.type = endedVoiceCall.duration > 0 ? "incoming" : "missed";
```

Comment:
No, when you have an incoming call, you get the popup with the green and red button. Green button = accept, timer starts running. Red button rejects the call (and sends it to voicemail), so call duration will be 0.